shell/platform/android/android_surface_gl.h

```diff
@@ -66,6 +66,10 @@ class AndroidSurfaceGL final : public GPUSurfaceGLDelegate,
   // |GPUSurfaceGLDelegate|
   sk_sp<const GrGLInterface> GetGLInterface() const override;
 
+  AndroidEGLSurface* GetOnscreenSurface() const {
```

Comment:
Please add a doc comment, probably explaining that this is only intended for tests.

e.g.

```
  // Obtain a raw pointer to the AndroidEGLSurface. This method is intended for use in tests. Callers must not delete this pointer.
```

Reply:
(I went ahead and made this change so we can just get this landed, rather than wait for another time zone cycle :)